x/incentive/types/claims.go

```diff
@@ -8,6 +8,19 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// RewardType is the id of some rewards
+type RewardType int
```

Comment:
@rhuairahrighairidh regarding

```
A potential downside: when all rewards use the same claim type, it would be harder to change specific rewards. eg adding some new feature to swap rewards, but not any others.
```

could this be used in the future to enable targeted behavior per reward type? E.g. allowing calling code to filter out non swap claims?